LEGO1/lego/legoomni/include/act3shark.h

```diff
@@ -3,9 +3,15 @@
 
 #include "legoanimactor.h"
 
-// VTABLE: LEGO1 0x100d7920
+/*
+	VTABLE: LEGO1 0x100d7920 LegoPathActor
+	VTABLE: LEGO1 0x100d79f0 LegoAnimActor
+*/
```

Comment:
Act3Shark doesn't have a vtable, even though it adds a new virtual function?

Reply:
It does, it extends LegoAnimActor non-virtually. I thought it was weird too, but it does in fact use the parent name. (You can verify the list of vtables in cvdump)